examples/multiple-css-and-js/index.js

```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const http = require('http');
+
+const Tailor = require('../../');
+
+const tailor = new Tailor({
+    templatesPath: __dirname + '/templates',
+    maxAssetLinks: 2
+});
+
+// Root Server
+http.createServer((req, res) => {
+    if (req.url === '/favicon.ico') {
+        res.writeHead(200, { 'Content-Type': 'image/x-icon' } );
+        return res.end('');
+    }
+    tailor.requestHandler(req, res);
+}).listen(8080, function () {
+    console.log('Tailor server listening on port 8080');
+});
+
+// Fragment server - Any http server that can serve fragments
+http.createServer((req, res) => {
```

Comment:
We can merge this PR once this is done, I have found a way for the performance hooks to work.